### DISCUSSION TITLE
play store?

### DISCUSSION THREAD

**Discussion by bmoorman4575** [luckydog7/Funkin-android#1729] [General]
Can we post mods to the google play store? 
If so is it only mods you make?

**Comment by ghost** [comment]
I mean just in general you shouldn't upload anything FNF related to App Stores. The only people that are doing it are scummy devs that try to profit off of something that's free but try to trick an idiot out there new to the fanbase or is possibly young and doesn't know better to download something and gain monetization.

**Reply by bmoorman4575** [reply]
ah ok then
